projects/ngx-prx-styleguide/src/lib/tags/tags.component.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { Component, DebugElement, NO_ERRORS_SCHEMA }    from '@angular/core';
+
+import { TagsComponent } from './tags.component';
+
+@Component({
+  selector: 'test-component',
+  template: `
+    <prx-tags
+      [selected]="testSelected"
+      [options]="testOptions"
+      (onChange)="setTestOutput($event)">
+    </prx-tags>
+  `
+})
+class TestComponent {
+  testSelected: any = [];
+  testOptions: any[] = [];
+  testOutput: any = undefined;
+  setTestOutput(val: any) { this.testOutput = val; }
+}
+
+describe('TagsComponent', () => {
+
+  let comp: TestComponent;
+  let fix: ComponentFixture<TestComponent>;
+  let de: DebugElement;
+  let el: HTMLElement;
+  let tags: TagsComponent;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [TestComponent, TagsComponent],
+      schemas: [ NO_ERRORS_SCHEMA ]
```

Comment:
Due to similarities with the SelectComponent, I followed the patterns used in that components spec. I can refactor to not require that schema.

Reply:
Ah, I didnt catch that. It might be that it required a lot of set up in the TestBed? Not that big of a deal if that's the case. That's definitely one thing that people tend to find tedious about Angular testing. Publish has some test helpers for that because things tend to need the same services set up.